ocs_ci/ocs/resources/objectbucket.py

```diff
@@ -383,9 +383,16 @@ def internal_delete(self):
             NotFoundError: In case the OBC was not found
 
         """
-        result = self.mcg.exec_mcg_cmd(f"obc delete {self.name}")
-        if "deleting" and self.name not in result.stderr.lower():
-            raise NotFoundError(result)
+        try:
+            result = self.mcg.exec_mcg_cmd(f"obc delete {self.name}")
+            if "deleting" and self.name not in result.stderr.lower():
```

Comment:
I see that it is from previous version, but could you fix it as follows:
```suggestion
            if "deleting" not in result.stderr.lower() and self.name not in result.stderr.lower():
```
because in the current version the first part of the condition is the `"deleting"` string, which is always `True`.

Reply:
I didn't touch the old condition since 2nd part is always False in current scenario. Anyway I updated to make sure "deleting" doesn't exists in stderr